kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
Yes but that is not why this was implemented in the first place, it's a 
quick workaround for stop/restart/reload, nothing more.

On 8/11/16 1:52 AM, Thijs Schreijer wrote:
>
> In kong/cmd/utils/log.lua 
> <https://github.com/Mashape/kong/pull/1486#discussion_r74387292>:
>
> > @@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
> >    end
> >  end
> >
> > +function _M.disable()
> > +  old_lvl = log_lvl
>
> point is; if someone messes up and forgets to reenable logging, the 
> whole thing goes silent, and it'll be hard to track down any issues
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Mashape/kong/pull/1486/files/4c1b1f6bb287c09c800f5413e654af5b6ccfa6ff#r74387292>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABEsN_8O7jPFzgDhav7sBt10k2aSCbc1ks5qeuLPgaJpZM4JhwBE>.
>